From efault@gmx.de Tue May 20 00:39:33 2008
Date: Tue, 20 May 2008 06:27:45 +0200
From: Mike Galbraith <efault@gmx.de>
To: Steven Rostedt <rostedt@goodmis.org>
Subject: Re: 2.6.25.4-rt2 (native_sched_clock() booboo)


On Mon, 2008-05-19 at 18:43 -0400, Steven Rostedt wrote:
> We are pleased to announce the 2.6.25.4-rt2 tree, which can be
> downloaded from the location:

I noticed a booboo watching the build.

---
 arch/x86/kernel/tsc_64.c |    1 +
 1 file changed, 1 insertion(+)

Index: linux-2.6.26/arch/x86/kernel/tsc_64.c
===================================================================
--- linux-2.6.26.orig/arch/x86/kernel/tsc_64.c
+++ linux-2.6.26/arch/x86/kernel/tsc_64.c
@@ -78,6 +78,7 @@ unsigned long long native_sched_clock(vo
 	rdtscll(a);
 	ret = cycles_2_ns(a);
 	preempt_enable_notrace();
+	return ret;
 }
 
 /* We need to define a real function for sched_clock, to override the
